samcli/lib/init/template_modifiers/cli_template_modifier.py

```diff
@@ -125,27 +126,10 @@ def _sanity_check(self) -> bool:
     def _print_sanity_check_error(self):
         pass
 
+    @abstractmethod
     def _write(self, template: list):
-        """
-        write generated template into SAM template
-
-        Parameters
-        ----------
-        template : list
-            array with updated template data
-        """
-        with open(self.template_location, "w") as file:
-            for line in template:
-                file.write(line)
-
-    def _get_template(self) -> List[str]:
-        """
-        Gets data the SAM templates and returns it in a array
+        pass
 
-        Returns
-        -------
-        list
-            array with updated template data
-        """
-        with open(self.template_location, "r") as file:
-            return file.readlines()
+    @abstractmethod
+    def _get_template(self) -> Any:
```

Comment:
We can remove the type hint here as it's an abstract method and doesn't really implement anything

Reply:
The type is different of XRayTemplate and AppInsightsTemplate. Thats why I used Any